### PR TITLE
do not map the extra cache drive on non virtualbox windows

### PR DESCRIPTION
**Pull request by mwrock**
…
Signed-off-by: Matt Wrock <matt@mattwrock.com>

### CODE REVIEW

File: lib/kitchen/driver/vagrant.rb [hunk 1]
```diff
@@ -359,15 +361,27 @@ def run_command(cmd, options = {})
...
+          if gem_home && (env["PATH"] || ENV["PATH"])
+            env["PATH"] ||= ENV["PATH"].dup if ENV["PATH"]
+            gem_bin = File.join(gem_home, "bin") + File::PATH_SEPARATOR
+            env["PATH"][gem_bin] = "" if env["PATH"].include?(gem_bin)
+          end
         end
 
         super(cmd, merged)
       end
+      # rubocop:enable Metrics/CyclomaticComplexity
```
Comment (afiune) [inline lib/kitchen/driver/vagrant.rb]: do we need this cop?

**Comment by mwrock** [comment]
The threshold could certainly be bumped IMO :) But I generally like this cop.

**Comment by afiune** [comment]
Tested on a Mac Host spinning up a Windows 7 Guest VM using VirtualBox:
```
➜  tk-windows git:(master) kitchen setup win-windows-7
-----> Starting Kitchen (v1.15.0-dev)
-----> Creating <win-windows-7>...
       Bringing machine 'default' up with 'virtualbox' provider...
       ==> default: Importing base box 'windows-7'...
==> default: Matching MAC address for NAT networking...
       ==> default: Setting the name of the VM: kitchen-tk-windows-win-windows-7_default_1480960736628_49942
       ==> default: Clearing any previously set network interfaces...
       ==> default: Preparing network interfaces based on configuration...
           default: Adapter 1: nat
       ==> default: Forwarding ports...
           default: 3389 (guest) => 3389 (host) (adapter 1)
           default: 5985 (guest) => 5985 (host) (adapter 1)
           default: 22 (guest) => 2222 (host) (adapter 1)
           default: 5986 (guest) => 55986 (host) (adapter 1)
       ==> default: Running 'pre-boot' VM customizations...
       ==> default: Booting VM...
       ==> default: Waiting for machine to boot. This may take a few minutes...
           default: WinRM address: 127.0.0.1:5985
           default: WinRM username: vagrant
           default: WinRM execution_time_limit: PT2H
           default: WinRM transport: negotiate
       ==> default: Machine booted and ready!
       ==> default: Checking for guest additions in VM...
           default: The guest additions on this VM do not match the installed version of
           default: VirtualBox! In most cases this is fine, but in rare cases it can
           default: prevent things such as shared folders from working properly. If you see
           default: shared folder errors, please make sure the guest additions within the
           default: virtual machine match the version of VirtualBox you have installed on
           default: your host and reload your VM.
           default:
           default: Guest Additions Version: 4.3.14
           default: VirtualBox Version: 5.0
       ==> default: Mounting shared folders...
           default: /omnibus/cache => /Users/salimafiune/.kitchen/cache
       ==> default: Machine not provisioned because `--no-provision` is specified.
       [WinRM] Established

       Vagrant instance <win-windows-7> created.
       Finished creating <win-windows-7> (1m52.41s).
```

**Comment by afiune** [comment]
cc/ @test-kitchen/maintainers ^^^^^^

**Pull request merged**